edmaths.sty

```diff
@@ -147,20 +153,9 @@
     \fi%  
 } {}
 
-
-
-% BODY HEIGHT
-% Set up the text body height. For the text height, we calculate the largest number of
-% lines that gives us no more than 4cm bottom margin. This depends on the line spacing,
-% which we need first.
-
+% Set the line spacing
 \setstretch{\stretchfactor}
 
-% Debug only: In the next line, \XXXfactor contains the number of lines on a text-only page.
-% \newlength{\XXXunit}\setlength{\XXXunit}{1pt}\newlength{\XXXfactor}\setlength{\XXXfactor}{1pt + ((672pt - \topskip) / \baselineskip) * \XXXunit}
-\newlength{\scratchlength}\setlength{\scratchlength}{\topskip + ((672pt - \topskip) / \baselineskip) * \baselineskip + \baselineskip}
-\setlength{\textheight}{\scratchlength}
-
```

Comment:
In the current version, the text height adjusts based on the line spacing, set by `stretchfactor`. Is this preserved with the update to using geometry or no?

Reply:
Yes, this is handled by `\geometry{heightrounded = true}` (which is likely to be more robust to other changes than the manual calculation).